presto-spi/src/main/java/com/facebook/presto/spi/statistics/StatisticsValue.java

```diff
@@ -0,0 +1,47 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.facebook.presto.spi.statistics;
+
+public final class StatisticsValue
+{
+    // todo eventually add some notion of statistic reliability
+    //      like estimator variance or confidence-interval.
+    //      Skipping for now as there hard to compute it properly and so far we do not have
+    //      usecase for that.
+
+    public static final double UNKNOWN_STATISTICS = Double.NaN;
+
+    private final double value;
```

Comment:
why is it double? And not a generic? You can leave it as it is, but for sure it will change.